app/models/photo.rb

```diff
@@ -2,16 +2,35 @@ class Photo < ActiveRecord::Base
   belongs_to :album
 
   has_attached_file :photo,
-                    :styles => {  :large => "640*480",  :medium => "300*300>",  :thumb => "100*100>" },
-                    :default_url => "/images/:style/missing.png",
-                    :path => ":rails_root/public/storage/#{Rails.env}#{ENV['RAILS_TEST_NUMBER']}/attachments/:id/:style/:basename.:extension",
-                    :url => "/storage/#{Rails.env}#{ENV['RAILS_TEST_NUMBER']}/attachments/:id/:style/:basename.:extension"
+                    styles: {
+                      large: "910*650>",
+                      medium: "700*500>",
+                      thumb: "350*250"
+                      },
+                    default_url: "/images/:style/missing.png",
+                    path: ":rails_root/public/storage/#{Rails.env}#{ENV['RAILS_TEST_NUMBER']}/attachments/:id/:style/:basename.:extension",
+                    url: "/storage/#{Rails.env}#{ENV['RAILS_TEST_NUMBER']}/attachments/:id/:style/:basename.:extension",
```

Comment:
Line is too long. [120/80]<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.